src/main/java/stream/flarebot/flarebot/commands/moderation/PermissionsCommand.java

```diff
@@ -307,6 +342,7 @@ public String getUsage() {
                 "`link <role>` - links the group to a discord role\n" +
                 "`unlink` - unlinks it from a role\n" +
                 "`list [page]` - lists the permissions this group has\n" +
+                "`massadd <role>` - puts everyone with the giving role into the group (works with @everyone and @here)\n" +
```

Comment:
I'd make this usages better, what about `massadd <@everyone/@here/role>`
Opinions @binaryoverload